daemon/delete.go

```diff
@@ -55,7 +57,15 @@ func (daemon *Daemon) ContainerRm(job *engine.Job) engine.Status {
 		if container.IsRunning() {
 			if forceRemove {
 				if err := container.Kill(); err != nil {
-					return job.Errorf("Could not kill running container, cannot remove - %v", err)
+					if err != syscall.ESRCH {
```

Comment:
There is same check in `container.Kill`, it can't return `syscall.ESRCH`. Maybe something changed since your PR.